js/app/src/Blocks.svelte

```diff
@@ -77,17 +80,27 @@
 	});
 
 	let params = new URLSearchParams(window.location.search);
-	let api_docs_visible = params.get("view") === "api";
-	const set_api_docs_visible = (visible: boolean) => {
-		api_docs_visible = visible;
+	let visible_side_panel: null | "api" | "traffic" = params.get("view") as
```

Comment:
The repeated `null | "api" | "traffic"` could be lifted into a `type SidePanelType = null | "api" | "traffic"`.